anyway/flask_app.py

```diff
@@ -1486,3 +1487,10 @@ class Cities(Resource):
     @api.doc("get get all cities")
     def get(self):
         return City.get_all_cities()
+
+
+@api.route("/api/news-flash/<int:id>", methods=["PUT"])
+class Set_Newsflash_Location_Qualification(Resource):
```

Comment:
Hi Shai, we use camel case notation for class names.

Reply:
done